cmd/kubeadm/app/cmd/config.go

```diff
@@ -115,6 +115,7 @@ func newCmdConfigPrintResetDefaults(out io.Writer) *cobra.Command {
 
 func newCmdConfigPrintActionDefaults(out io.Writer, action string, configBytesProc func() ([]byte, error)) *cobra.Command {
 	kinds := []string{}
+	deprecatedMsg := fmt.Sprintf("Print the default value of component config API objects is deprecated for action: %s, it will be ingored if set and will be removed in a future release.", action)
```

Comment:
typo: ignore.

Reply:
Done.

print it to stdout use `fmt.Fprintln` instead.